docker/start-op-challenger.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+set -e
+
+# how do we want to handle network-dependant configs here?
+DISPUTE_GAME_FACTORY=0x5a20cD16d4E51D3B2A6b2Ab460Ac735Fe421b820
+L2_OUTPUT_ORACLE=0xE6Dfba0953616Bacab0c9A8ecb3a9BBa77FC15c0
```

Comment:
still not sure how to deal with these...

Reply:
Should we just switch them out based on the `$NETWORK` variable like we do when starting up op-geth?

Reply:
Yes that can work!

Reply:
@ncitron It turns out the dispute game factory contract is still in development and not deployed on Goerli yet.
So for now I used a random address just to make the program start without parsing errors